test/unit/test_assembly.py

```diff
@@ -103,7 +103,7 @@ def test_empty_input_succeed(self):
 
         # run refine_assembly
         args = [inFasta, inBam, outFasta, "--chr_names", 'G5012.3', "--min_coverage", '3', "--novo_params",
-                "-r Random -l 30 -g 40 -x 20 -t 502"]
+                "-r Random -l 30 -g 40 -x 20 -t 502 -c {}".format(_CPUS)]
         args = assembly.parser_refine_assembly(argparse.ArgumentParser()).parse_args(args)
         print(args)
```

Comment:
Remove

Reply:
So, if a novoalign.lic file exists (and it does in Travis), I think Novoalign defaults to use all cores. For unit tests, we want it to default to _CPUS (which varies depending on whether xdist is enabled).